config/confighttp/confighttp_test.go

```diff
@@ -208,6 +208,48 @@ func TestDefaultHTTPClientSettings(t *testing.T) {
 	assert.EqualValues(t, 90*time.Second, *httpClientSettings.IdleConnTimeout)
 }
 
+func TestProxyURL(t *testing.T) {
+	testCases := []struct {
+		desc        string
+		proxyURL    string
+		expectedURL *url.URL
+	}{
+		{
+			desc:        "default config",
+			expectedURL: nil,
+		},
+		{
+			desc:        "proxy is set",
+			proxyURL:    "http://proxy.example.com:8080",
+			expectedURL: &url.URL{Scheme: "http", Host: "proxy.example.com:8080"},
+		},
```

Comment:
Can you please add a case with an invalid proxyURL string to get 100% test coverage?

Reply:
@dmitryax I am travelling till the Wed without a laptop so it will be bit hard.